modules/sdk-test-ts/src/index.ts

```diff
@@ -1015,3 +1015,8 @@ spacetimedb.reducer(
     }
   }
 );
+
+const X = t.object('X', { a: t.i32() });
+spacetimedb.view('a', X, () => {
+  return { a: 1 };
+});
```

Comment:
It's a little bare bones I must say haha

Reply:
Oh, whoops, I didn't really mean to leave that in.